ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -4,6 +4,9 @@ server {
 
     charset     utf-8;
 
+    server_tokens off;
+    more_set_headers 'Server: Kitware ;)';
```

Comment:
It would be good to add some documentation here. Also would it affect if we deploy it on aws?